libxxhash.pc.in

```diff
@@ -1,10 +1,11 @@
 #   xxHash - Extremely fast hash algorithm
+#   Copyright (C) 2012-2020, Yann Collet, Facebook
 #   BSD 2-Clause License (https://www.opensource.org/licenses/bsd-license.php)
 
 prefix=@PREFIX@
 exec_prefix=${prefix}
```

Comment:
In theoty, someone cound define EXEC_PREFIX different than PREFIX. The libdir in the .pc file would then evaluate wrong. (Not a common case though).